relay/pkg/btc/btc.go

```diff
@@ -0,0 +1,42 @@
+package btc
+
+import (
+	"math/big"
+
+	"github.com/ipfs/go-log"
+)
+
+var logger = log.Logger("relay-btc")
+
+// Header represents a Bitcoin block header.
+type Header struct {
+	// TODO: implementation
+}
+
+// Client exposes methods needed to interact with the BTC blockchain.
+// It's defined as an interface in order to allow swapping the actual
+// implementation. For example, during the integration tests, it can
+// be more convenient to use a mock instead of the real network.
+type Client interface {
+	// TODO: implementation
+}
```

Comment:
Not sure if I follow what the `Client` interface will be used for. Do we have any analogy in other keep clients?

Reply:
No, we don't have an analogy. The idea is to have a possibility to swap actual client implementation within `btc.Chain`. For example, for regular use, it can be `rpc.Client` which actually connects to the BTC network. On the other hand, a `local.Client` can be used as a mock blockchain client while running a simple integration test.

Reply:
For clarification, I have not created `btc.Chain` as an interface because in contrary to the `pkg/chain` host chains we will always have BTC as the source chain. The only thing which may change here is whether we use an actual RPC client or a mocked one for testing purposes. This is why I think the `btc.Client` abstraction is right in that context.

Reply:
I'd rather stick with `bitcoin.Chain` interface TBH. Bitcoin is a chain and we'll be extracting information from the chain and pushing it to a host chain. `bitcoin.Chain` may consist of multiple interfaces and pass those individual scoped interfaces to functions - we can then implement tests easier and have functions scoped to this part of the interface they really need. 

Client feels like a low-level concept to me that lives under the chain interface and is responsible for the communication with an external node, that's also how Ethereum and Celo implementations work.

We can have `Connect() bitcon.Chain` and `ConnectLocal() bitcoin.Chain`, we do not expect more than two options here.

Reply:
Ok, done in https://github.com/keep-network/tbtc/commit/37d083999a579af04bff35ae6cafb20f6ac46ebc